### PR TITLE
[13.0][IMP] mrp_unbuild_cust_alu_analytics: new field company_id for mrp.unbuild.process.type model

### PR DESCRIPTION
**Pull request by adriresu**

### CODE REVIEW

File: mrp_unbuild_cust_alu_analytics/views/mrp_unbuild_process_type_views.xml [hunk 1]
```diff
@@ -14,6 +14,7 @@
                             </group>
                             <group>
                                 <field name="bom_line_ids" widget="many2many_tags"/>
+                                <field name="company_id" groups="base.group_multi_company" options="{'no_create': True}"/>
```
Comment (ChristianSantamaria) [inline mrp_unbuild_cust_alu_analytics/views/mrp_unbuild_process_type_views.xml:17]: Does it make sense that this field can be edited by the user?
```suggestion
                                <field name="company_id" groups="base.group_multi_company" readonly="1"/>
```

**Review by ChristianSantamaria — COMMENTED** [review]
Tested, work correctly, miss squash the commits and i have a question...

File: mrp_unbuild_cust_alu_analytics/views/mrp_unbuild_process_type_views.xml (hunk 1 above)
Comment (dalonsod) [reply mrp_unbuild_cust_alu_analytics/views/mrp_unbuild_process_type_views.xml:17]: It makes sense, see https://github.com/solvosci/slv-manufacture/pull/71#discussion_r1580746898

**Comment by dalonsod** [comment]
Last changes made:
* Attended pending comment https://github.com/solvosci/slv-manufacture/pull/71#discussion_r1589371694
* Proper and more accurate PR title.

**Comment by dalonsod** [comment]
Tested in AluTest environment :+1:

**Pull request merged**